integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/AbstractIntegrationTest.java

```diff
@@ -242,56 +228,24 @@ protected void createEnvironmentWithFreeIpa(TestContext testContext) {
                     .withNetwork()
                     .withTelemetry("telemetry")
                     .withTunnel(Tunnel.CLUSTER_PROXY)
-                    .withCreateFreeIpa(Boolean.FALSE)
+                    .withCreateFreeIpa(Boolean.TRUE)
+                    .withFreeIpaImage(commonCloudProperties().getImageValidation().getFreeIpaImageCatalog(),
+                            commonCloudProperties().getImageValidation().getFreeIpaImageUuid())
```

Comment:
Are these lines always setup together? If yes, I think would be better as a dedicated method, like `enableFreeIpaCreation` or stuff like this

Reply:
And I would hold the original `.withCreateFreeIpa(Boolean.TRUE)` method as well

Reply:
the original logic is that we are creating freeipa always, but after environment creation was success without actual freeipa, which does not make any sense from product point of view

Reply:
discussed internally, we can review/remove/refactor image settings of freeipa within different task